portal/src/assets/styles/forms.css

```diff
@@ -303,3 +303,41 @@
     }
   }
 }
+
+.signature-widget-form {
```

Comment:
When using tailwind, we want to write as little styles in css files as possible, and instead use the classes directly on the components. Typically we only want to writes styles in css files only when these styles are going to be reused in a lot of places, or to implement things that we can't do with tailwind classes.

So for these styles, you can create a class to add to the radio input to handle the background image (and maybe its related sizing and positioning), but otherwise we should move the rest of these styles to be directly on the elements if possible.

For the text input, you can add the class `field` to its wrapping element, because that class is defined for styling those inputs.